ops/ansible/roles/bfd-server/templates/bfd-server.sh.j2

```diff
@@ -16,6 +16,7 @@ DB_PASSWORD='{{ data_server_db_password }}'
 DB_CONNECTIONS_MAX='{{ data_server_db_connections_max }}'
 BFD_V2_ENABLED='{{ data_server_v2_enabled }}'
 BFD_PAC_ENABLED='{{ pac_resources_enabled }}'
+BFD_CLAIM_SOURCE_TYPES='{{ pac_claim_source_types }}'
```

Comment:
Wondering about name here; could we include PACA in there? (BFD_PACA_CLAIM_SOURCE_TYPES)